### DISCUSSION TITLE
Purchasing a Canokey in Europe

### DISCUSSION THREAD

**Discussion by stv0g** [canokeys/canokey-product#21] [General]
Hi all,

do you know where I could purchase a Canokey?

**Comment by cnZary** [comment]
Thank you for your interest in purchasing a Canokey. Currently, our products are available for sale exclusively on Taobao. However, if you're looking to buy within the European Union, please contact me directly at zary@canokeys.com, and we can discuss the best way to facilitate your purchase.

**Reply by stv0g** [reply]
Thanks a lot @cnZary.

I now understood that the commercial version (Canonkeys Pidgeon) has a locked bootloader and is not that useful for my testing: I am developing [hawkes](https://github.com/cunicu/hawkes) a Golang implementation of Noise-based handshakes using which is backed by ECDH on security tokens.

I will go ahead with the nRF dongle instead.

**Reply by cnZary** [reply]
Thanks for your interest! if you ever need any assistance or have questions about our products in the future, feel free to reach out.
